x-pack/plugins/actions/server/saved_objects/migrations.ts

```diff
@@ -161,7 +178,48 @@ const addisMissingSecretsField = (
   };
 };
 
-function pipeMigrations(...migrations: ActionMigration[]): ActionMigration {
-  return (doc: SavedObjectUnsanitizedDoc<RawAction>) =>
+const moveRelatedSavedObjects = (
```

Comment:
This is handled in https://github.com/elastic/kibana/pull/108964 so I would remove this

Reply:
Thanks! I will wait till you PR will be merged and fix my changes over that.